src/components/backend-ai-multi-select.ts

```diff
@@ -0,0 +1,262 @@
+/**
+ @license
+ Copyright (c) 2015-2022 Lablup Inc. All rights reserved.
+ */
+
+import {css, CSSResultGroup, html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+import {
+  IronFlex,
+  IronFlexReverse,
+  IronFlexAlignment,
+  IronFlexFactors,
+  IronPositioning
+} from '../plastics/layout/iron-flex-layout-classes';
+import {BackendAiStyles} from './backend-ai-general-styles';
+
+import '@material/mwc-list/mwc-check-list-item';
+import '@material/mwc-icon-button/mwc-icon-button';
+import '@material/mwc-list/mwc-list';
+
+import '../plastics/lablup-shields/lablup-shields';
+
+/**
+ Backend.AI Multi Select
+
+ `backend.ai-multi-select` is multi-selectable UI
+
+ Example:
+
+ <backend-ai-multi-select></backend-ai-multi-select>
+
+ @group Backend.AI Web UI
+ @element backend-ai-multi-select
+ */
+
+@customElement('backend-ai-multi-select')
+export default class BackendAIMultiSelect extends LitElement {
+  public shadowRoot: any; // ShadowRoot
+  @property({type: Object}) comboBox;
```

Comment:
Do we need to expose the `comboBox` as an element's property? It seems like it is only used internally, and then, I think it is better to declare it as a private class variable.

Reply:
I agree, there's no reason to expose those items since it's only for "superadmin". I'll change it.